vue3_ts/src/views/veevalidate-yup/EventsExample.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div class="max-w-450 mx-auto my-30">
+    <h1 class="mb-30 text-center">Events example</h1>
+
+    <transition-group name="slide-fade">
+      <form v-if="submitCount < 3" @submit="onSubmit">
```

Comment:
native form submit will reload a page, please use .prevent modifier

Reply:
in the "onSubmit" method I'm running "handleSubmit" which I took from "useForm" and it prevents native form behavior by it own.
Perhaps I will add a comment to make it clear, what do you think?